lib/rubocop/cop/lint/debugger.rb

```diff
@@ -72,8 +52,26 @@ def message(node)
           format(MSG, source: node.source)
         end
 
-        def binding_irb?(node)
-          binding_irb_call?(node)
+        def with_receiver?(node)
+          !!node.receiver
+        end
+
+        def special_rule?(node)
+          return false unless node.receiver.is_a?(RuboCop::AST::ConstNode)
+          return true if node.receiver.const_name == 'Kernel' && node.method?(:save_and_open_page)
```

Comment:
In particular it's very weird we hardcoded this.

Reply:
I also don't like it. Look above why I did it.